i2c/zynq/libzynq-i2c.c

```diff
@@ -427,12 +445,14 @@ static int i2c_initClk(void)
 
 static int i2c_reset(void)
 {
+	const i2c_info_t *info = &devsInfo[i2c.devID];
+
 	platformctl_t pctl;
 	int err;
 
 	pctl.action = pctl_get;
 	pctl.type = pctl_devreset;
-	pctl.devreset.dev = pctl_ctrl_i2c_rst;
+	pctl.devreset.dev = info->rst;
 	err = platformctl(&pctl);
```

Comment:
Pls fix while editing this
```suggestion
	return platformctl(&pctl);
```

Reply:
fixed

Reply:
Pls check, it is as it was

Reply:
I guess the comment was attached to the wrong line
![obraz](https://github.com/user-attachments/assets/4588cb2a-39b6-4ccc-aaf9-791353572b15)